.vscode/settings.json

```diff
@@ -1,7 +1,18 @@
 {
   "editor.formatOnSave": true,
   "git.ignoreLimitWarning": true,
-  "cSpell.words": ["PALEWHITE", "Multichain", "testid", "iban", "IBAN"],
+  "cSpell.words": [
+    "PALEWHITE",
+    "Multichain",
+    "testid",
+    "iban",
+    "IBAN",
+    "onlineid",
+    "videoid",
+    "noreferrer",
+    "OLKILUL",
+    "Bahnhofstrasse"
```

Comment:
These two seem to be unnecessary 🤔

Reply:
there are in our stories and i saw warnings therefore I have added them